### PR TITLE
BOAC-2874: Simplifies passenger manifest checkbox logic

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2874
https://jira.ets.berkeley.edu/jira/browse/BOAC-2882

I think it makes sense for the checkboxes to work like an inclusive OR. Thoughts?

### CODE REVIEW

**Comment by lyttam** [comment]
From Steven's comments it sounds like his main use case will be searching for a particular user in order to edit them - in which case we don't want the checkboxes to prevent that user from showing up in a UID or name search.

File: src/components/admin/Users.vue [hunk 1]
```diff
@@ -287,7 +287,7 @@ export default {
       const permissionsMatch = adminMatch || advisorMatch || canvasAccessMatch || directorMatch || dropInAdvisorMatch || schedulerMatch || noPermissionsMatch;
       const statusMatch = activeMatch || deletedMatch || blockedMatch || expiredMatch;
 
-      return nameUidMatch && deptMatch && permissionsMatch && statusMatch;
+      return nameUidMatch && deptMatch && (permissionsMatch || statusMatch);
```
Comment (johncrossman) [inline src/components/admin/Users.vue]: @lyttam - IMO, this is less intuitive. As a dev who uses the feature to find test-users, the feature is good enough in its current state. I think another opinion would be useful.  cc: @paulkerschen

Comment (paulkerschen) [reply src/components/admin/Users.vue]: If I'm reading this right, replacing AND with OR means that we couldn't use the checkboxes in combination to say "Show me all active advisors" or "Show me all active admins," which is the kind of thing I often do in development. There's some ugliness no matter how we tackle it, but I'd also vote for keeping the current arrangement as good enough.

Comment (lyttam) [reply src/components/admin/Users.vue]: That's how I would use this feature too (e.g. to find test users).  I'll close this until someone has a better idea.

**Pull request closed without merging**